app/app.py

```diff
@@ -13,11 +13,15 @@
 from .models import update_db_from_github
 
 app = Flask(__name__)
+app.debug = True
```

Comment:
So, this forces the app into debug mode always, including production?  Since it's an internal tool, that sounds fine, just want to make sure I understand.